.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
       args: ["--config", ".github/configurations/python_linters/.flake8"]
 
 -   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Please change to this version in the pipeline too, otherwise there would be a version missmatch